parsing_2.py

```diff
@@ -108,6 +108,14 @@ def main():
             max_requests = requests_dict[request]
             max_requests_name = request
 
+    #Question 6: same as above, but looking for lower than min_requests
+    min_requests = 2000
```

Comment:
This could be larger:
```suggestion
    min_requests = float("inf")
```